config/initializers/meilisearch.rb

```diff
@@ -1,4 +1,4 @@
 MeiliSearch::Rails.configuration = {
   meilisearch_url: Rails.env.local? ? "http://localhost:7700" : "http://91.107.208.207:7700", # example: http://localhost:7700
-  meilisearch_api_key: ENV["MEILI_MASTER_KEY"]
+  meilisearch_api_key: Rails.env.local? ? nil : ENV["MEILI_MASTER_KEY"]
 }
```

Comment:
Thanks I will add this directly to main as it could create issues for users not having a correct local key